modules/misc/converter.py

```diff
@@ -3,19 +3,45 @@
 # Matteo Golin (linguini1)
 
 
+def milli_degrees_to_celsius(milli_degrees: int) -> float:
+    """Returns the passed milli degrees Celsius value in Celsius as a float.
+    Args:
+        milli_degrees (int): The temperature in milli degrees Celsius
+
+    Returns:
+        float: The temperature in Celsius"""
+
+    return round(milli_degrees / 1000, 2)
+
+
 def celsius_to_fahrenheit(celsius: float) -> float:
-    """Returns the passed Celsius value in Fahrenheit as a float."""
+    """Returns the passed Celsius value in Fahrenheit as a float.
+    Args:
+        celsius (float): The temperature in  Celsius
```

Comment:
No worries about adding the types in the docstring; those are apparent in the actual type annotations so omitting them will avoid duplication.